pkg/network/multus.go

```diff
@@ -36,6 +48,35 @@ func changeSafeMultus(prev, next *opv1alpha1.NetworkAddonsConfigSpec) []error {
 	return nil
 }
 
+// Currently not implemented, since there are no obsolete objects under this module
+func CleanUpMultus(ctx context.Context, client k8sclient.Client, objs []*unstructured.Unstructured) []error {
+	// Get existing
```

Comment:
Please add subfunction per each case of additional cleanup. This one should be for the renaming. This function should have proper TODO to drop it once we don't support the old version anymore.